model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -61,25 +63,20 @@ internal class LicenseResolver(private val ortResult: OrtResult) {
         }
 
         val matchedFindingsWithoutExcludedCopyrights = matchedFindings.map { finding ->
-            if (project != null) {
-                val copyrights = finding.copyrights.mapNotNullTo(sortedSetOf()) { copyrightFindings ->
-                    val locations = copyrightFindings.locations.filterTo(sortedSetOf()) {
-                        pathExcludes.none { exclude -> exclude.matches(it.getRelativePathToRoot()) }
-                    }
-                    if (locations.isNotEmpty()) copyrightFindings.copy(locations = locations)
-                    else null
+            val copyrights = finding.copyrights.mapNotNullTo(sortedSetOf()) { copyrightFindings ->
+                val locations = copyrightFindings.locations.filterTo(sortedSetOf()) {
+                    pathExcludes.none { exclude -> exclude.matches(it.getRelativePathToRoot()) }
                 }
-                finding.copy(copyrights = copyrights)
-            } else {
-                finding
+                if (locations.isNotEmpty()) copyrightFindings.copy(locations = locations)
+                else null
             }
+            finding.copy(copyrights = copyrights)
```

Comment:
Nit: Blank line before this one.

Reply:
Agree, but that's out of scope of this change.

Reply:
Uhm, it's not out of scope since you've introduced that line. And so far our policy was that reviewing commit-by-commit is fine, and even intermediate issues should be addressed. If you want to change that policy, we should discuss that in the team beforehand.

Reply:
I have not introduced that line. Please take a look at it again.

Reply:
Well, ok, you *moved* it and changed its indentation, so you already modified its formatting.

Reply:
Can you please indicate if your requested change is mandatory?

Reply:
I don't get why you prefer to spend the time asking this instead of spending the same amount of time just doing it, but no, it's not mandatory.

Reply:
The answer to that is quite simple, I want to do things in a way which make most sense to me!
And for doing so I just need the information whether a change request is mandatory or not.

In case I prefer to do things differently as requested I will keep on asking whether it's mandatory or not. I think always providing such indicator in the first place would be quite beneficial to our discussion. How about you?

Reply:
I *try* to give that indication, but sometimes I don't because I'm yet unsure myself whether I regard a change as mandatory or not, or because I simply forget about it. In any case I try to minimize the time spent by *all* parties on review discussions, and do whatever all parties can agree on and takes the least time to get done.

Reply:
That's a valid point, sometimes it is just not clear whether something is mandatory, e.g. it might need clarification first. We could still make such indicator mandatory and have a special indicator for the case where its not clear. In one of my previous projects we used e.g. -2,-1,0, +1, +2 as mandatory prefix for all comment where 0 denoted the unclear case. IMO these prefixes had a great impact on efficiency of the reviews. Anyhow, I guess we should just discuss this with the whole team. I'll bring it up in the caht again soon.